tracing/src/main/java/com/palantir/remoting1/tracing/Tracer.java

```diff
@@ -61,16 +61,18 @@ public static void initTrace(Optional<Boolean> isObservable, String traceId) {
     }
 
     /**
-     * Opens a new span for this thread's call trace, labeled with the provided operation and parent span.
+     * Opens a new span for this thread's call trace, labeled with the provided operation and parent span. Only allowed
+     * when the current trace is empty.
      */
-    public static OpenSpan startSpan(String operation, String parentSpanId) {
+    public static OpenSpan startSpan(String operation, String parentSpanId, Optional<SpanType> type) {
```

Comment:
I'm not a fan of Optional as argument and tend to agree with most of https://stuartmarks.wordpress.com/2016/09/27/vjug24-session-on-optional/